grammar.js

```diff
@@ -1146,6 +1148,25 @@ module.exports = grammar(C, {
       $.literal_suffix
     ),
 
+    alternative_unary: $ => prec.left(PREC.UNARY, seq(
+      field('operator', 'not'),
+      field('argument', $._expression)
+    )),
+
+    alternative_binary: $ => {
```

Comment:
How about adding those alternative operators to unary_expression / binary_expression rules inherited from the C grammar? I guess having the same parse tree could simplify some queries.

Perhaps we could even add them directly to the C grammar? C95 has these defined in a header file as preprocessor macros, see: https://en.wikipedia.org/wiki/C_alternative_tokens